lib/basedriver/driver.js

```diff
@@ -276,6 +280,11 @@ class BaseDriver extends Protocol {
         // have an unexpected shutdown event, for example, we can cancel it from
         // outside, rejecting the current command immediately
         this.curCommandCancellable = new B((r) => { r(); }).then(() => { // eslint-disable-line promise/prefer-await-to-then
+          // if one of the args is an image element, handle it separately
+          const imgElId = getImgElFromArgs(args);
```

Comment:
this looks a bit hacky

Reply:
hmm, it was the only way I could think of to pass control to ImageElement in the case of an image element existing in the arguments. I agree it's not super pretty. Any other ideas?